### PR TITLE
Refer to original source metadata in source XML and JSON

### LINKED ISSUE #120: Refer to original source metadata files in source XML and JSON
As implemented with #104, currently the original source metadata files (manifest.safe, annotation folder) are just copied to the `source` directory of a processed scene. The source metadata files in OGC XML and STAC JSON format, which are generated by the processor should include references to all of the original files. This ensures that programs can completely traverse the available metadata files and check for completeness.

### PR DESCRIPTION
**Pull request by maawoo**
closes #120 

Here is a preview of how the files are listed in the source XML and JSON files:
image

### CODE REVIEW

File: S1_NRB/nrb.py [hunk 1]
```diff
@@ -329,12 +331,10 @@ def format(config, product_type, scenes, datadir, outdir, tile, extent, epsg, wb
     
     # copy support files
     schema_dir = os.path.join(S1_NRB.__path__[0], 'validation', 'schemas')
-    support_dir = os.path.join(ard_dir, 'support')
     schemas = os.listdir(schema_dir)
-    os.makedirs(support_dir, exist_ok=True)
     for schema in schemas:
         schema_in = os.path.join(schema_dir, schema)
-        schema_out = os.path.join(support_dir, schema)
+        schema_out = os.path.join(os.path.join(ard_dir, 'support'), schema)
```
Comment (johntruckenbrodt) [inline S1_NRB/nrb.py:337]: double `os.path.join`
```python
schema_out = os.path.join(ard_dir, 'support', schema)
```

**Review by johntruckenbrodt — CHANGES_REQUESTED** [review]
Looks good @maawoo. I only found a very minor thing.
Could you upload some example files here so I can have a look?

**Comment by maawoo** [comment]
[Here](https://upload.uni-jena.de/data/65041564ea2064.00247820/S1B_IW_ORB__1SDV_20211114T190503_029588_0387F4_28RDS_A80B.zip) you go (accessible until 1 week from now)

File: S1_NRB/nrb.py (hunk 1 above)
Comment (maawoo) [reply S1_NRB/nrb.py]: Whoops, fixed https://github.com/SAR-ARD/S1_NRB/pull/136/commits/43575a4e12562a2372e3c85e26b53d8d9ea01766

**Comment by johntruckenbrodt** [comment]
@maawoo what happens if there are two source scenes? Won't there be double keys in the metadata?

**Comment by maawoo** [comment]
> @maawoo what happens if there are two source scenes? Won't there be double keys in the metadata?

The XML and JSON files are generated for each source scene individually. So duplicate keys shouldn't be an issue.
Feel free to check this example: https://upload.uni-jena.de/data/6508122e960422.78001587/S1B_IW_ORB__1SDV_20211114T190517_029588_0387F4_28RDT_7A8C.zip

**Comment by johntruckenbrodt** [comment]
Of course. Monday morning I guess 😅.

**Review by johntruckenbrodt — APPROVED** [review]

**Pull request merged**